src/app/about/about.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
need to create components folder and all the components you need to add in that folder